third-party/linshare/src/main/java/org/apache/james/linshare/LinshareConfiguration.java

```diff
@@ -78,8 +78,8 @@ public static LinshareConfiguration from(Configuration configuration) throws Mal
 
     @VisibleForTesting
     LinshareConfiguration(URL url, AuthorizationToken token) {
-        Preconditions.checkNotNull(url, "'" + URL + "' can not be null");
-        Preconditions.checkNotNull(token, "'" + TOKEN + "' can not be null");
+        Preconditions.checkNotNull(url, String.format("'%s' can not be null", URL));
```

Comment:
I think we are able to write this like this: 

```
Preconditions.checkNotNull(url, "'%s' can not be null", URL);
```

(I just discovered it!)

This could lead us to a way larger refactoring ;-)

Reply:
Yes, it is good idea.